services/cpu_unit_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest import mock
+from cpu_service import calculate_cpu_usage, get_cpu_usage_array, _collect_cpu_usage,fake_mem
+
+
+class CPUMonitorTestCase(unittest.TestCase):
+    @mock.patch('cpu_monitor.ts.get_history_time')
```

Comment:
This won't work! Because the `patch` need to find the code, and it's doing that by a relative import. For example, this would work better:
```python
@mock.patch('services.time_service.get_history_time')
@mock.patch('psutil.cpu_percent'
```

The import is not from the current file